README.md

```diff
@@ -80,7 +79,26 @@ information on changes between releases.
 
 ## Getting involved and contributing!
 
-Are you interested in contributing to kops? We the maintainers and community would love your suggestions, contributions, and help! We have a quickstart guide on [adding a feature](/docs/development/adding_a_feature.md) and the maintainers can be contacted at any time to learn more about how to get involved.
+Are you interested in contributing to kops? We, the maintainers and community, would love your suggestions, contributions, and help! We have a quick-start guide on [adding a feature](/docs/development/adding_a_feature.md).  Also, the maintainers can be contacted at any time to learn more about how to get involved.
+
+In the interest of getting more new folks involved with kops, we are starting to tag issues with `good-starter-issue`. These are typically issues that have smaller scope but are good ways to start to get acquainted with the codebase.
+
+We also encourage ALL active community participants to act as if they are maintainers, even if you don't have "official" write permissions. This is a community effort, we are here to serve the Kubernetes community. If you have an active interest and you want to get involved, you have real power! Don't assume that the only people who can get things done around here are the "maintainers".
+
+We also would love to add more "official" maintainers, so show us what you can do!
+
+What this means:
+
+__Issues__
+* Help read and triage issues, assist when possible.
+* Point out issues that are duplicates, out of date, etc.
+  - Even if you don't have tagging permissions, make a note and tag maintainers. (`/close`,`/dupe #127`)
+
+__Pull Requests__
+* Read and review the code. Leave comments, questions, and critiques (`/lgtm` )
+* Download, compile, and run the code and make sure it does what the PR says and doesn't break something else
```

Comment:
I would suggest getting rid of the 

> doesn't break something else

language. I think I would rather see something along the lines of

> Download, compile, and run the code and make sure the tests pass (`make test`). Also verify that the new feature seems sane, follows best architectural patterns, and includes tests.